ccdb/law/tests/test_views.py

```diff
@@ -31,9 +36,71 @@ def test_empty(self):
         self.assertEquals(r.status_code, 302)
 
     def test_real(self):
-        self.s = Snapshot.objects.create(
-            label="test snapshot",
-            status="vetted")
+        self.s = SnapshotFactory()
         r = self.c.get("/autocomplete/?term=nothing")
         self.assertEquals(r.status_code, 200)
         self.assertEquals(r.content, '[]')
+
+
+class TestViewCharge(TestCase):
+    def setUp(self):
+        self.c = Client()
```

Comment:
Just a note: by default, TestCase has a self.client attribute. Is there any need to create another?